src/System Application/App/Rest Client/app.json

```diff
@@ -34,11 +34,11 @@
   "idRanges": [
     {
       "from": 2350,
-      "to": 2361
+      "to": 2362
     }
   ],
   "target": "OnPrem",
-  "runtime": "12.0",
+  "runtime": "14.0",
```

Comment:
Please remove the runtime version. We do not define it for Microsoft owned code.

Reply:
Runtime has been removed from app.json